catalog/rulesets.json

```diff
@@ -168,6 +168,30 @@
     },
     {
       "ghrepo": "coinbase/rules_ruby"
+    },
+    {
+      "ghrepo": "buildfoundation/bazel_rules_detekt"
+    },
+    {
+      "ghrepo": "DataDog/rules_oci_bootstrap"
+    },
+    {
+      "ghrepo": "acqio/rules_microsoft_azure"
+    },
+    {
+      "ghrepo": "spietras/rules_conda"
+    },
+    {
+      "ghrepo": "ubiquitoustech/rules_vite"
```

Comment:
oh @dymart what do you want to do with this one??

Reply:
Ya I can update rules_vite to the new rules_js :)